Library/Homebrew/cmd/audit.rb

```diff
@@ -72,7 +72,7 @@ def audit
       fa.audit
 
       next if fa.problems.empty?
-
+      fa.problems
```

Comment:
yeah that's dead code. it used to be fa.problems.uniq! which is no longer needed. thanks for spotting it.